frontend/src/pages/org/archived-item-qa/archived-item-qa.ts

```diff
@@ -818,9 +818,11 @@ export class ArchivedItemQA extends TailwindElement {
                 placement="bottom-start"
               >
                 <btrix-button
-                  icon
-                  variant=${!this.splitView ? "primary" : "neutral"}
+                  raised
+                  ?filled=${!this.splitView}
+                  size="small"
                   @click="${() => (this.splitView = !this.splitView)}"
+                  class="m-0.5"
                 >
                   <sl-icon name="vr" label=${msg("Split view")}></sl-icon>
```

Comment:
I don't think this label is required, the tooltip content is already read aloud (twice actually, which is a different issue I think) as the important information here is the description of the control, not the description of what the icon looks like.

```suggestion
                  <sl-icon name="vr"></sl-icon>
```

What _isn't_ read aloud here is the button's toggle state... Because _technically_ this should probably be a checkbox?  Complicated!

Reply:
Ahhh, right. Hmm, lemme see what I can do

Reply:
From Discord convo, the right move here is with `role="switch"` and updating `aria-checked` or `aria-pressed`

Reply:
I think `aria-pressed` is the most applicable since we're using a single button and not a checkbox or switch. The role remains `button` in that case, too. https://w3c.github.io/aria/#aria-pressed